errors.go

```diff
@@ -21,6 +21,31 @@ type ErrorsPayload struct {
 	Errors []*ErrorObject `json:"errors"`
 }
 
+// Code represents the application-specific error code, expressed as a string value.
+type Code string
+
+// UnmarshalJSON converts the JSON (number or strign) code into string Code
+func (c *Code) UnmarshalJSON(data []byte) error {
+	var rawV interface{}
+	if err := json.Unmarshal(data, &rawV); err != nil {
+		return err
+	}
+	switch v := rawV.(type) {
+	case string:
+		*c = Code(v)
+	case float64:
+		*c = Code(fmt.Sprintf("%.0f", v))
+	default:
+		return fmt.Errorf("json: cannot unmarshal into Go value of type %T ", v)
+	}
+	return nil
+}
+
+// ToString returns code as a string
+func (c *Code) ToString() string {
```

Comment:
Not sure if this function is useful. But if it is I think it needs to become
```
func (c Code) ToString() string {
```
pointer is unnecessary

Reply:
That is a bit overkill I agree, I will keep it simple for now and just remove this method